### ISSUE TITLE
How to include Reagent Bank

### ISSUE THREAD

**Issue by RickB72169** [nikkisaurus/FarmingBar#51]
Unless I've missed it, I don't see an option to include items in the reagent bank in the Farming Bar; it only shows a count for, in this example, ores that are in my bags. I searched through the issues already listed, and the one that mentioned this issue seemed to be caused by Datastore. I disabled Datastore and the issue persisted. When I hover my cursor over an ore in the bar, there is an entry that says "Include Bank", and next to it "1/1". When I move the ore to the reagent bank, the counter goes to 0.

**Comment by nikkisaurus** [comment]
I'm having trouble replicating the issue you're having. Could you give me more specifics?

Is this happening with anything you move to and from the reagent bank?
Or is it a specific item?
Can you share the objective? (If you have an objective template, just click export and copy and paste the code. If not, control right click the button, click save as template, then find that objective and export it).
How are you moving the item? Manually placed or using the deposit all reagents button.

Basically dumbing it down step by step can be helpful so I can try to run into the error myself. With my testing, I'm receiving the correct count regardless or which bag or bank slot items are in.

**Comment by RickB72169** [comment]
This issue has occurred with both ore and herbs, all from the Dragon Isles.
As far as moving the items, I use the Deposit all reagents, from both the Farming Bar and from the option available in the reagent bank.
I haven't set up an objective; I  was using the Farming Bar much as I did with Farmit: to keep track of my inventory in my bags and my reagent bank.
I also am using Adibags, but I also tested with Adibags disabled, and the issue persisted.

**Comment by nikkisaurus** [comment]
I might have figured out what's going on and why I can't replicate the issue. I believe you haven't actually selected the option to include the item in the bank, but the tooltip is incorrectly telling you that you have.

To include the bank, control right click the button to open the objective editor. Under the trackers tab, you'll see a list of trackers. If you are only using it by dragging items onto the bar, there should only be 1. When you select this tracker, there is a section called "Include" with a checkbox labeled "Bank". Make sure this is selected. With DataStore, this is also where you can enable counts from your alts or guild bank.

Once this is selected, see if the issue persists and then let me know. In the meanwhile, I'll be working on a fix for the incorrect display.

**Comment by RickB72169** [comment]
Thank you for the help; it is greatly appreciated.  Your instructions fixed the issue for me.
